### ISSUE TITLE
Coverage dependence not written for StickingCoefficient reactions

### ISSUE THREAD

**Issue by rwest** [ReactionMechanismGenerator/RMG-Py#2270]
Apparently if you have `StickingCoefficient` kinetics with `coverage_dependence` attributes, coming from a reaction library, the coverage dependence does not get written to the chemkin file (or at least not in such a way that they end up in the cantera file, which is what we actually looked at).

**Comment by ChrisBNEU** [comment]
at least when I tried it, it does get written to the chemkin file like this: 
```
! Reaction index: Chemkin #1; RMG #1
! Library reaction: Surface/CPOX_Pt/Deutschmann2006_adjusted
! Flux pairs: H2(2), H*(10); X(1), H*(10); X(1), H*(10); 
X(1)+X(1)+H2(2)=H*(10)+H*(10)                       4.600e-02 0.000     0.000    
    STICK
    COV / X(1)                                      0         -1        0.000     /
```

but it does not get written to the cantera file: 
```
#  Reaction index: Chemkin #1; RMG #1
#  Library reaction: Surface/CPOX_Pt/Deutschmann2006_adjusted
#  Flux pairs: H2(2), H*(10); X(1), H*(10); X(1), H*(10);
# surface1 Reaction 1
surface_reaction('X(1) + X(1) + H2(2) <=> H*(10) + H*(10)', stick(4.600000e-02, 0.0, 0.0),
                 id='surface1-1')
```

I suspect it has something to do with ck2cti parser. when I execute the following code with the chemkin files: 
```python3
from cantera import ck2cti
parser = ck2cti.Parser()
parser.convertMech(
    'meoh/chemkin/chem_annotated-gas.inp', 
    outName='test_cantera.cti', 
    quiet=True, 
    permissive=True, 
    surfaceFile='meoh/chemkin/chem_annotated-surface.inp'
)
```
I get the same result in cantera. using a later version of cantera does not fix it. however, using ck2yaml gives me the following: 
```yaml
surface1-reactions:
- equation: X(1) + X(1) + H2(2) <=> H*(10) + H*(10)  # Reaction 1
  sticking-coefficient: {A: 0.046, b: 0.0, Ea: 0.0}
  coverage-dependencies:
    X(1): [0.0, -1.0, 0.0]
```

**Comment by rwest** [comment]
Maybe we should be making yaml not cti anyway.

**Comment by ChrisBNEU** [comment]
resolved with merge of #2288
